.github/workflows/process_new_pull_request.yml

```diff
@@ -0,0 +1,230 @@
+            // REVIEWER ASSIGNMENT
+            // ======================
+            if (hasIndividualReviewers) {
+              await github.rest.issues.addAssignees({
+                owner,
+                repo,
+                issue_number: prNumber,
+                assignees: requestedIndividualReviewers
+              });
+
+              const firstReviewer = requestedIndividualReviewers[0];
+              const assignmentMessage = `Assigning @${firstReviewer} for the first pass review of this PR. Thanks!`;
+              
+              if (assignmentMessage.length <= 500) {
+                await github.rest.issues.createComment({
+                  owner,
+                  repo,
+                  issue_number: prNumber,
+                  body: assignmentMessage
+                });
+              } 
+            }
+
+            // ======================
+            // NOTIFICATION SYSTEM
+            // ======================
+            if (!hasIndividualReviewers) {
+              // Notify if no individual reviewers exist (even if team reviewers exist)
+              // This helps catch cases where team has no active reviewers
+              
+              // Try to assign a random member from dev-workflow-reviewers team
+              try {
+                const teamMembers = await github.rest.teams.listMembersInOrg({
+                  org: owner,
+                  team_slug: 'dev-workflow-reviewers'
+                });
+                
+                if (teamMembers.data && teamMembers.data.length > 0) {
+                  // Pick a random reviewer from the team
+                  const randomIndex = Math.floor(Math.random() * teamMembers.data.length);
+                  const randomReviewer = teamMembers.data[randomIndex].login;
+                  
+                  if (validateGitHubUsername(randomReviewer)) {
+                    // Add as reviewer and assignee
+                    await github.rest.pulls.requestReviewers({
+                      owner,
+                      repo,
+                      pull_number: prNumber,
+                      reviewers: [randomReviewer]
+                    });
+                    
+                    await github.rest.issues.addAssignees({
+                      owner,
+                      repo,
+                      issue_number: prNumber,
+                      assignees: [randomReviewer]
+                    });
+                    
+                    const assignmentMessage = `No individual reviewers found. Assigned @${randomReviewer} from the dev-workflow-reviewers team. Thanks!`;
+                    
+                    await github.rest.issues.createComment({
+                      owner,
+                      repo,
+                      issue_number: prNumber,
+                      body: assignmentMessage
+                    });
+                    
+                    return; // Skip the general notification since we've assigned someone
```

Comment:
this will also skip the new contributor guidance, which we don't want. Instead, can we put the general notification logic in the `catch` block?